packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -662,6 +722,9 @@ export class ApexDebug extends LoggingDebugSession {
     } else {
       response.success = true;
     }
+    if (response.success) {
+      this.clearIdleTimers();
```

Comment:
Clear idle timers on session disconnect.

Reply:
I think this should be wrapped in a finally, i.e. we want to clear the timers in any case. Also, what happens is the response ain't successful? Shouldn't this be considered a dead session anyway and the timers cleared no matter what?